liana-gui/src/loader.rs

```diff
@@ -377,6 +327,65 @@ impl Loader {
     }
 }
 
+fn get_bitcoind_log(log_path: PathBuf) -> impl Stream<Item = Option<String>> {
+    channel(1, move |mut output| async move {
```

Comment:
```suggestion
    channel(5, move |mut output| async move {
```
I'd be "conservative w/ a size of `5`